src/mainwindow.cpp

```diff
@@ -2166,7 +2167,7 @@ void MainWindow::maximizeWindow()
 
     } else {
         setWindowState(windowState() | Qt::WindowMaximized);
-        //        setMargins(QMargins(0,0,0,0));
+        setMargins(QMargins(0, 0, 0, 0));
```

Comment:
I'll just note that `QMargins()` already "Constructs a margins object with all margins set to 0.", so I would suggest to leave out the `0, 0, 0, 0` everywhere.

Reply:
Good catch.